lib/segment/src/telemetry.rs

```diff
@@ -151,7 +151,7 @@ impl TelemetryOperationAggregator {
             let from = if i < SLIDING_WINDOW_LEN {
                 0
             } else {
-                SLIDING_WINDOW_LEN - i
+                i - SLIDING_WINDOW_LEN
```

Comment:
do you have a test for this or you just noticed the issue by chance?

Reply:
It was my mistake from prev PR. Prev PR I tested with small amount of collected data and didn't catch this bug